presto-product-tests/src/main/resources/sql-tests/testcases/window_functions/rankNoPartitionImplicitFrame.sql

```diff
@@ -0,0 +1,2 @@
+-- database: presto; groups: window;
```

Comment:
How it's different than _denseRankNoPartitionImplicitFrame.sql_ despite different function (`dense_rank` vs `rank`)?